app/src/main/java/ch/epfl/sdp/contamination/Neighbor.java

```diff
@@ -0,0 +1,57 @@
+package ch.epfl.sdp.contamination;
+
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * Neighbor is a simple type of Carrier, who has a FIXED status (hence also without status history)
```

Comment:
Not at all.
Layman is a type of Carrier with many features like the possibility of being observed and of locally storing the history of infection probability.
All this is not needed (since it's not provided) for the Carriers returned by DataReceiver. That class should only provides simple Carriers with immutable status and probability and without any history. That role is played by Neighbors